test/files/pos/u000a.scala

```diff
@@ -0,0 +1,11 @@
+// Unicode newline in a single-line comment?
+// Compiler will expect code on the line.
+// Here the code is valid.
+
+/* \n object foo */
+// \n object foo
```

Comment:
I'm not convinced arbitrary escapes are relevant, as they are specific to contexts like ordinary string literals and interpolators (where they have specific, arbitrary semantics).